xrootd/resources/xrootd-origin.cfg

```diff
@@ -93,3 +93,6 @@ xrd.trace {{.Logging.OriginXrd}}
 cms.trace {{.Logging.OriginCms}}
 xrootd.tls all
 scitokens.trace {{.Logging.OriginScitokens}}
+{{if .Xrootd.ConfigFile}}
+continue {{.Xrootd.ConfigFile}}
```

Comment:
Up to you, but it might be nice to do some validation on this beforehand so we can move the error domain out of XRootD and into Pelican. I know how much we all love debugging XRootD's cryptic ramblings ;) A basic check might look for:
```
- If a file path is provided, does it exist?
- If a file path is provided, does Pelican have permissions to read it?
```

A more advanced check might provide explicit information about anything that the provided config file overwrites (as opposed to adds to).

Reply:
Another thing that would be helpful is to add a debug statement somewhere that mentions a custom config is being used and the path to the config. That way more of the useful information makes it into the log and requires less actual knowledge of the `pelican.yaml`.

Reply:
I like the error checking idea and I indeed __love__ debugging with XRootD.

Reply:
> A more advanced check might provide explicit information about anything that the provided config file overwrites (as opposed to adds to).

This is going to be very tricky as XRootD has its own policy at each directive level on overwrite/appending, which is not explicitly exposed anywhere (besides the doc)